### PR TITLE
Sub - category update

### PR DESCRIPTION
**Pull request by kirakenchin**
Updated sub categories to be able to classify all the guides easily

### CODE REVIEW

**Comment by nonkerdoob** [comment]
Avoid punctuations like spaces, commas and dots in file names
``getting started`` to ``getting-started``
``display, sound and keyboard`` to ``display-sound-keyboard``

these appear in the url so have to be free from special characters
also making it smaller can make it easier to remember for quick linking in chats and Reddits, grammar is not important.

Every category folder must have an accompanying ``.md`` file with the same name as the category folder.

I,e,
```
+-- ..
|-- getting-started.md
|-- getting-started
|    |-- no-os-legion.md 
|    |-- posts.md ....
|-- 
+--

An attempt has been made to explain it here https://laptopwiki.github.io/contribute/create-post/ but evidently it needs work lmao. Better guide coming soon

They accompanying ``.md`` file must contain these front matter and heading
```markdown
---
layout: wiki
title: 
description: (if any)
toc_content: false
parent: parent title (if any)
grand-parent: grand-parent title (if any)
ancestor: ancestor title (if any)
---

# Heading
```
so for ``getting-started`` folder, the ``getting-started.md`` file will look like this

TBD

**Comment by nonkerdoob** [comment]
> oh right, i missed the key points, will recheck and update it under this or other pull request

aah, no worries
feel free to ask as many questions you like

**Comment by nonkerdoob** [comment]
``parent: `` and ``grand-parent: `` should match the ``title: `` of the parent posts. Currently it is set to ``getting-started`` and ``basics``.

Open ``getting-started.md`` and ``basics.md`` and copy the ``title: `` field.

Changes should like this , they are case sensitive
image

**Comment by kirakenchin** [comment]
image
 
Here, grand parent is _guides iirc. so Grand parent should not be mentioned right? since the file is not inside getting-started folder? or the grand parent should always be mentioned based on which folder it is based of? in this case getting-started

**Comment by nonkerdoob** [comment]
> Here, grand parent is _guides iirc. so Grand parent should not be mentioned right? since the file is not inside getting-started folder? or the grand parent should always be mentioned based on which folder it is based of? in this case getting-started

If the grand-parent is a section, which in this case is _guides.
It is not needed (sections have an underscore in front of their folder names)

Also, basics.md doesn't need to be a parent of itself 😆 
it doesn't need a parent field. Unless we are going to make a general category. in which case, the parent will be the title of the general.md file.

File: _guides/basics/basics.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ layout: wiki
 title: Basics
 nav_order: 1
 toc_content: false
+parent: Basics
```
Comment (nonkerdoob) [inline _guides/basics/basics.md:6]: This is not needed. 
Undo changes.

File: _guides/basics/getting-started/custom-res.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+layout: wiki
+title: Add 60Hz Refresh Rate
+description: "Adds 60Hz to monitors/laptops with no 60Hz refresh rate"
+parent: getting-started
+grand parent: basics
```
Comment (nonkerdoob) [inline _guides/basics/getting-started/custom-res.md:6]: ``parent: Getting Started``
``grand-parent: Basics``

Have to make changes for fields with this error.

Comment (nonkerdoob) [inline _guides/basics/getting-started/custom-res.md:6]: ``parent: Getting Started``
``grand-parent: Basics``

Have to make changes for fields with this error.

File: _guides/basics/basics.md (hunk 1 above)
Comment (nonkerdoob) [reply _guides/basics/basics.md]: Removed ``parent: `` 
Issu resolved.

**Comment by kirakenchin** [comment]
i will add this pic here, since it is a bit complicated due to multiple words and punctuations
image

check if this right too pls

**Pull request merged**